### PR TITLE
Added divider component

### PR DESCRIPTION
**Pull request by 3rfan**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

A `Divider` component

#### Why?

To have an indicator to distinguish different page sections.

#### Example Usage

~~~js
<Divider>I am the mighty Divider</Divider>
~~~

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/Divider.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    children: string,
+};
+
+export default class Divider extends React.PureComponent<Props> {
+    render() {
+        const {children} = this.props;
+        const dividerClass = classNames(
+            dividerStyles.divider,
+            {
+                [dividerStyles.noText]: !children,
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/Divider.js]: How about using the [`empty` pseudo selector](https://developer.mozilla.org/de/docs/Web/CSS/:empty) instead?

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/divider.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    &::after {
+        content: "";
+        display: inline-block;
+        flex: 1;
+        height: 1px;
+        background: $dividerBackgroundColor;
+    }
+
+    &::before {
+        margin-right: 80px;
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/divider.scss]: 40px

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/divider.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    &::after {
+        margin-left: 80px;
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/divider.scss]: 40px

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/divider.scss (hunk 2 above)
Comment (danrot) [reply src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/divider.scss]: I have also seen 80px and 20px in the design 🙈

**Pull request merged**